libraries/SITL/SITL.cpp

```diff
@@ -291,6 +291,13 @@ const AP_Param::GroupInfo SITL::var_info3[] = {
     AP_GROUPEND
 };
 
+// fourth table of user settable parameters for SITL.
+const AP_Param::GroupInfo SITL::var_info4[] = {
+    AP_GROUPINFO("BATT_TEMP",      1, SITL, batt_temperature, 20.0),
+
```

Comment:
Why not move the existing batt parameters here:
  AP_GROUPINFO("BATT_VOLTAGE",  19, SITL,  batt_voltage,  12.6f),
    AP_GROUPINFO("BATT_CAP_AH",   20, SITL,  batt_capacity_ah,  0),

Reply:
A.) I was avoiding conversions.... on this one
B.) The VARInfo table for SITL is actually full.... and I wasn't sure at the time where to varInfo4 at besides an oldindex at the top....
C.) I think how parameters work in SITL needs to be rethought or moved into individual cpp files as is best possible.  Doing that would be the best time for conversion personally.